tests/testDynamicBrcode.js

```diff
@@ -0,0 +1,61 @@
+const assert = require('assert');
+const starkbank = require('../index.js');
+const generateExampleDynamicBrcodesJson = require('./utils/dynamicBrcode.js').generateExampleDynamicBrcodesJson;
+
+starkbank.user = require('./utils/user').exampleProject;
+
+describe('TestDynamicBrcodePost', function () {
+    this.timeout(10000);
+    it('test_success', async () => {
+        let brcodes = generateExampleDynamicBrcodesJson(5);
+        brcodes = await starkbank.dynamicBrcode.create(brcodes);
+        for (let brcode of brcodes) {
+            assert(typeof brcode.uuid == 'string');
+        }
+    });
+});
+
+describe('TestDynamicBrcodeQuery', function () {
+    this.timeout(10000);
+    it('test_success', async () => {
+        let i = 0;
+        const brcodes = await starkbank.dynamicBrcode.query({ limit: 5 });
+        for await (let brcode of brcodes) {
+            assert(typeof brcode.uuid == 'string');
+            i += 1;
+        }
+        assert(i === 5);
+    });
+});
+
+describe('TestDynamicBrcodeQueryGet', function () {
+    this.timeout(10000);
+    it('test_success', async () => {
+        let brcodes = await starkbank.dynamicBrcode.query({ limit: 1 });
+        for await (let brcode of brcodes) {
+            assert(typeof brcode.uuid == 'string');
+            brcode = await starkbank.dynamicBrcode.get(brcode.uuid);
+            assert(typeof brcode.uuid == 'string');
```

Comment:
Não é melhor se certificar que o uuid não é nulo do que o seu tipo?

Reply:
Serve pras outras validações que estão assim

Reply:
é q se for uma string ele já n é nulo, ai já estaria testando os dois